flink-python/pyflink/table/tests/test_udf.py

```diff
@@ -204,6 +204,28 @@ def eval(self, col):
             self.t_env.register_function(
                 "non-callable-udf", udf(Plus(), DataTypes.BIGINT(), DataTypes.BIGINT()))
 
+    def test_udf_without_arguments(self):
+        @udf(input_types=[], result_type=DataTypes.BIGINT(), deterministic=True)
```

Comment:
1. If there is no arguments, is it possible not declare input_types?
2. What about using lambda expression instead? Then just one line for each udf

Reply:
1. In current design it will throw an exception if we don't declare input_types. And IMO the lack of input_types should not be considered as a no-argument UDF. It means we should infer the input types when the UDF is called. 
2. I have replaced current UDFs with lambda in the latest commit.